projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -75,6 +75,12 @@ public function get_item_permissions_check( $request ) { // phpcs:ignore Generic
 	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
 	 */
 	public function get_item( $request ) { // phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter, VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			require_once WP_CONTENT_DIR . '/mu-plugins/masterbar/admin-menu/load.php';
+		} else {
+			require_once JETPACK__PLUGIN_DIR . '/modules/masterbar/admin-menu/load.php';
+		}
```

Comment:
We're getting to a point where we have quite a few classes, and where part of me wonders if in a v2, we shouldn't move the whole thing to a Composer package so we could rely on autoloading, and require the package here, on WordPress.com, and maybe in wpcomsh so we could update that package on Atomic before the new version of Jetpack ships.

I think it may be worth discussing in the future.